modules/mod_ginger_rdf/support/dbpedia.erl

```diff
@@ -75,16 +75,21 @@ get_resource(Uri, Language) ->
 %% @doc The nl dbpedia is missing the thumbnails of entries. Fetch the thumbnail from
 %% the wikipedia page.
 get_resource_thumbnail(Triples) ->
-    case fetch_wikipedia_url(Triples) of
-        undefined ->
-            undefined;
-        WikipediaUrl ->
-            case z_url_metadata:fetch(WikipediaUrl) of
-                {ok, MD} ->
-                    z_url_metadata:p(image, MD);
-                {error, _} ->
-                    undefined
-            end
+    case ?DEBUG(fetch_wikipedia_url(Triples)) of
```

Comment:
Is it correct that there should be a DEBUG here?

Reply:
oops :-)